src/main/webapp/WEB-INF/pages/manage/overview_metadata-bootstrap.ftl

```diff
@@ -52,6 +52,18 @@
                         <@s.submit cssClass="btn btn-sm btn-outline-success" name="edit" key="button.edit"/>
                     </#if>
                 </form>
+                <form action='replace-eml.do' method='post' enctype="multipart/form-data" style="margin-top: 10px;">
```

Comment:
I'd suggest having this form first, before 'metadata-basic.do' to align with the section 'Source Data'.
Let's also replace an inline style with bootstrap classes

Reply:
Do you mean having something like that ?

![image](https://user-images.githubusercontent.com/56630013/114910686-7c817b00-9e1e-11eb-8479-9f834e2bd331.png)

It will be more similar to the Source section, but maybe too much different for current users... ?